server/models/StudiesModel/index.js

```diff
@@ -10,6 +10,32 @@ const StudiesModel = {
       .filter((study) => !STUDIES_TO_OMIT.includes(study))
       .sort((prevStudy, nextStudy) => (prevStudy < nextStudy ? -1 : 1))
   },
+  index: async (db, query) =>
```

Comment:
`index` isn't a model method, it is a controller method. Similarly, `all` isn't a controller method, it's a model method.

Reply:
[b6abb0b](https://github.com/AMP-SCZ/dpdash/pull/723/commits/b6abb0bd1e5a49faa249a852a52ea7280c83be87)